src/app/cic/controllers/root.js

```diff
@@ -14,6 +14,19 @@ class RootController extends BaseController {
         req.sessionModel.set("dateOfBirth", sharedClaims.birthDate[0].value);
       }
     }
+
+    try {
+      const headers = {
+        "x-govuk-signin-session-id": req.session.tokenId
+      }
+
+      const { data } = await req.axios.get(`${API.PATHS.SESSION_CONFIG}`, { headers });
+      req.sessionModel.set("journeyType", data.journey_type);
+    } catch (error) {
+      console.log("Error fetching journey type", error)
```

Comment:
Is this left in for the purpose of unit testing? Assuming cause this is middleware it'll only be in ECS logs not dev console

Reply:
Yeah left it in so we can check ECS for logs relating to just this in case we need to debug. Find it so difficult to trawl through and look for "error"